src/features/annotation-thread/AnnotationThreadContent.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+import React from 'react';
+import flow from 'lodash/flow';
+import ActivityThread from '../../elements/content-sidebar/activity-feed/activity-feed/ActivityThread';
+import AnnotationActivity from '../../elements/content-sidebar/activity-feed/annotations';
+import API from '../../api/APIFactory';
+import commonMessages from '../../elements/common/messages';
+import LoadingIndicator from '../../components/loading-indicator/LoadingIndicator';
+import { withAPIContext } from '../../elements/common/api-context';
+import withErrorHandling from '../../elements/content-sidebar/withErrorHandling';
```

Comment:
We're introducing a lot of dependencies on modules from the `elements` directory, where typically we see the inverse, with the top-level Elements using some components from the `features` directory.